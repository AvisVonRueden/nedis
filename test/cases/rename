*3
$3
SET
$3
foo
$9
something
--------
*3
$6
RENAME
$3
foo
$6
foobar
--------
*2
$3
GET
$3
foo
--------
*2
$3
GET
$6
foobar
