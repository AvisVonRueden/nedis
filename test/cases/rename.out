+OK
+OK
$-1
$9
something
:1
